<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>493</width>
    <height>524</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>493</width>
    <height>524</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>汉王智远人脸锁LOGO生成工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>240</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>LOGO</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>161</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="UPtoolButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>向上</string>
    </property>
   </widget>
   <widget class="QToolButton" name="DowntoolButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>向下</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>人脸锁屏幕（320x240）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>动画列表</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>381</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>动画横坐标：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="x_pos_lineEdit"/>
     </item>
     <item row="2" column="0" colspan="10">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="y_pos_lineEdit_2"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>动画纵坐标：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7" colspan="3">
      <widget class="QLineEdit" name="w_len_lineEdit_3"/>
     </item>
     <item row="0" column="5" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>动画宽度：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="h_len_lineEdit_4"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>动画长度：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="time_lineEdit_5"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>动画间隔：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>添加背景</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>添加动画</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>删除已选动画</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5" colspan="3">
      <widget class="QPushButton" name="build_pushButton">
       <property name="text">
        <string>开始演示</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>生成文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
